python_ipware/python_ipware.py

```diff
@@ -337,7 +333,7 @@ def get_best_ip(
             return best_client_ip, True
 
         # the incoming ips match our proxy count
-        if self.proxy_count > 0 and proxy_count_validated:
+        if self.proxy_count and self.proxy_count > 0 and proxy_count_validated:
```

Comment:
Here I would just check `if self.proxy_count is not None and proxy_count_validated` because self.proxy_count = 0 now also means that the proxy_count did go through validation. If I change this, I have to adapt the tests `test_proxy_count_zero_dont_care_proxy_pass` and `test_proxy_count_zero_exact_zero_proxy_pass`.

Reply:
Great catch! thx